.github/workflows/benchmark_x86_tdx.yml

```diff
@@ -108,7 +108,7 @@ jobs:
       max-parallel: 1
     timeout-minutes: 60
     container: 
-      image: asterinas/asterinas:0.16.0-20250802-tdx
+      image: asterinas/asterinas:0.16.0-20250802
```

Comment:
Leave this change out of this PR because the QEMU installed in this Docker image is still incompatible with TDX.